source/objects/lander.lua

```diff
@@ -417,8 +417,6 @@ function Lander.draw()
 		love.graphics.draw(ship.image, x,y, math.rad(lander.angle), sx, sy, ox, oy)
 
 		-- draw flames
-		local ox = flame.width / 2
-		local oy = flame.height / 2
```

Comment:
This local variable seems to be important.

Reply:
That was my mistake, I've now reverted it back. :)